### PR TITLE
Add a document to describe how to define experiment scope

### LINKED ISSUE #677: Add a document to describe how to define experiment scope
## Feature Request

**Is your feature request related to a problem? Please describe:**

We need a document to describe how to define the experiment scope

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve?

Close #677

### CODE REVIEW

File: website/docs/user_guides/experiment_scope.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+id: experiment_scope
+title: Define the Scope of Chaos Experiment
+sidebar_label: Define the Scope of Chaos Experiment
+---
+
+This document describes how to define the chaos experiment scope.
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:7]:
```suggestion
This document describes how to define the scope of a chaos experiment.
```

File: website/docs/user_guides/experiment_scope.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Chaos Mesh provides a lot of selectors for users and you can use them to define the scope of our chaos experiment.
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:9]:
```suggestion
Chaos Mesh provides a variety of selectors, which you can use to define the scope of your chaos experiment. 
```

File: website/docs/user_guides/experiment_scope.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+---
+
+This document describes how to define the chaos experiment scope.
+
+Chaos Mesh provides a lot of selectors for users and you can use them to define the scope of our chaos experiment. 
+These selectors are defined in the `spec.selector` field of the chaos object.
+
+## Namespace selectors
+
+Namespace selectors are used to filtering the chaos experiment targets by the namespaces and defined as a set of string. If this selector is empty,
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:14]: Please be consistent in using the plural form. Namespace selectors or selector?

File: website/docs/user_guides/experiment_scope.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+This document describes how to define the chaos experiment scope.
+
+Chaos Mesh provides a lot of selectors for users and you can use them to define the scope of our chaos experiment. 
+These selectors are defined in the `spec.selector` field of the chaos object.
+
+## Namespace selectors
+
+Namespace selectors are used to filtering the chaos experiment targets by the namespaces and defined as a set of string. If this selector is empty, 
+Chaos Mesh will use the namespace of the chaos experiment object as the default namespace selector. For example:
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:15]:
```suggestion
The default namespace selector for Chaos Mesh is the chaos experiment object. For example:
```

File: website/docs/user_guides/experiment_scope.md (hunk 3 above)
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:14]:
```suggestion
Namespace selectors filter the chaos experiment targets by the namespace. Defined as a set of strings.
```

File: website/docs/user_guides/experiment_scope.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+```yaml
+spec: 
+  selector:
+    namespaces:
+      - "app-ns"
+```
+
+## Label selectors
+
+Label selectors are used to filtering the chaos experiment targets by the labels and defined as a map of string keys and values. For example:
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:26]:
```suggestion
Label selectors filter chaos experiment targets by the label. Defined as a map of string keys and values. For example:
```

File: website/docs/user_guides/experiment_scope.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+```yaml
+spec: 
+  selector:
+    labelSelectors:
+      "app.kubernetes.io/component": "tikv"
+```
+
+## Annotation selectors
+
+Annotation selectors are used to filtering the chaos experiment targets by the annotations and defined as a map of string keys and values. For example:
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:37]:
```suggestion
Annotation selectors filter chaos experiment targets by the annotation. Defined as a map of string keys and values. For example:
```

File: website/docs/user_guides/experiment_scope.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+```yaml
+spec: 
+  selector:
+    annotationSelectors:
+      "example-annotation": "group-a"
+```
+
+## Field selectors 
+
+Field selectors are used to filtering the chaos experiment targets by the resource fields and defined as a map of string keys and values.
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:48]:
```suggestion
Field selectors filter chaos experiment targets by the resource field. Defined as a map of string keys and values. 
```

File: website/docs/user_guides/experiment_scope.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+spec: 
+  selector:
+    annotationSelectors:
+      "example-annotation": "group-a"
+```
+
+## Field selectors 
+
+Field selectors are used to filtering the chaos experiment targets by the resource fields and defined as a map of string keys and values. 
+This selector lets you select Kubernetes resources based on the value of one or more resource fields. For example:
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:49]: ``` This selector lets you select Kubernetes resources based on the value of one or more resource fields. ```

This is not necessary. It's repeating what's been said before

File: website/docs/user_guides/experiment_scope.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+This selector lets you select Kubernetes resources based on the value of one or more resource fields. For example: 
+
+```yaml
+spec: 
+  selector:
+    fieldSelectors:
+      "metadata.name": "my-pod"
+```
+
+More details about field selectors, you can refer to the [Kubernetes document](https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/).
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:58]:
```suggestion
For more details about field selectors, refer to the [Kubernetes document](https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/).
```

File: website/docs/user_guides/experiment_scope.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+
+
+## Pod phase selectors
+
+Pod Phase selectors are used to filtering the chaos experiment targets by the condition and defined as a set of string. 
+Supported condition: `Pending`, `Running`, `Succeeded`, `Failed`, `Unknown`. For example:
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:64]:
```suggestion
Supported conditions: `Pending`, `Running`, `Succeeded`, `Failed`, `Unknown`. For example: 
```

File: website/docs/user_guides/experiment_scope.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+    fieldSelectors:
+      "metadata.name": "my-pod"
+```
+
+More details about field selectors, you can refer to the [Kubernetes document](https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/).
+
+
+## Pod phase selectors
+
+Pod Phase selectors are used to filtering the chaos experiment targets by the condition and defined as a set of string.
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:63]:
```suggestion
Pod Phase selectors filter chaos experiment targets by the condition. Defined as a set of string. 
```

File: website/docs/user_guides/experiment_scope.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+```yaml
+spec: 
+  selector:
+    podPhaseSelectors:
+      - "Running"
+```
+
+## Pod selectors
+
+Pod selectors specify specific pods and defined as a map of string keys and a set values. The key of this map defines the namespace which pods belong to,
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:75]:
```suggestion
Pod selectors filter chaos experiment targets by the pod. Defined as a map of string keys and values. The key in this map specifies the namespace which the pods belong to, 
```

File: website/docs/user_guides/experiment_scope.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+spec: 
+  selector:
+    podPhaseSelectors:
+      - "Running"
+```
+
+## Pod selectors
+
+Pod selectors specify specific pods and defined as a map of string keys and a set values. The key of this map defines the namespace which pods belong to, 
+and each value is a set of pod names. If this selector is not empty, these pod defined in this map are used directly and ignore the other selectors. For example:
```
Comment (dcalvin) [inline website/docs/user_guides/experiment_scope.md:76]:
```suggestion
and each value under the key is a pod. If this selector is not empty, these pod defined in this map are used directly and other defined selectors will be ignored. For example: 
```

File: website/docs/user_guides/kernel_chaos.md [hunk 14]
```diff
@@ -44,7 +44,7 @@ For more sample files, see [examples](https://github.com/pingcap/chaos-mesh/tree
 Description:
 
 * **mode** defines the mode to select pods.
-* **selector** specifies the target pods for chaos injection.
+* **selector** specifies the target pods for chaos injection. For more details, see: [Define the Scope of Chaos Experiment](experiment_scope.md).
```
Comment (dcalvin) [inline website/docs/user_guides/kernel_chaos.md:47]:
```suggestion
* **selector** specifies the target pods for chaos injection. For more details, see [Define the Scope of Chaos Experiment](experiment_scope.md).
```

File: website/docs/user_guides/network_chaos.md [hunk 15]
```diff
@@ -47,7 +47,7 @@ Description:
 
 * **action** defines the specific chaos action for the pod. In this case, it is network partition.
 * **mode** defines the mode to run chaos action.
-* **selector** specifies the target pods for chaos injection.
+* **selector** specifies the target pods for chaos injection. For more details, see: [Define the Scope of Chaos Experiment](experiment_scope.md).
```
Comment (dcalvin) [inline website/docs/user_guides/network_chaos.md:50]:
```suggestion
* **selector** specifies the target pods for chaos injection. For more details, see [Define the Scope of Chaos Experiment](experiment_scope.md).
```

File: website/docs/user_guides/pod_chaos.md [hunk 16]
```diff
@@ -46,7 +46,7 @@ Description:
 * **mode** defines the mode to run chaos action. Supported mode: `one` / `all` / `fixed` / `fixed-percent` / `random-max-percent`.
 * **value** depends on the value of `mode`. If `mode` is `one` or `all`, leave `value` empty. If `fixed`, provide an integer of pods to do chaos action. If `fixed-percent`, provide a number from 0 to 100 to specify the percent of pods the server can do chaos action. If `random-max-percent`, provide a number from 0 to 100 to specify the max percent of pods to do chaos action.
 * **duration** defines the duration for each chaos experiment. The value of the `duration` field is `30s`, which indicates that pod failure will last 30 seconds.
-* **selector** is used to select pods that are used to inject chaos actions.
+* **selector** is used to select pods that are used to inject chaos actions. For more details, see: [Define the Scope of Chaos Experiment](experiment_scope.md).
```
Comment (dcalvin) [inline website/docs/user_guides/pod_chaos.md:49]:
```suggestion
* **selector** is used to select pods that are used to inject chaos actions. For more details, see [Define the Scope of Chaos Experiment](experiment_scope.md).
```

File: website/docs/user_guides/time_chaos.md [hunk 17]
```diff
@@ -38,7 +38,7 @@ For more sample files, see [examples](https://github.com/pingcap/chaos-mesh/tree
 Description:
 
 * **mode** defines the mode to select pods.
-* **selector** specifies the target pods for chaos injection.
+* **selector** specifies the target pods for chaos injection. For more details, see: [Define the Scope of Chaos Experiment](experiment_scope.md).
```
Comment (dcalvin) [inline website/docs/user_guides/time_chaos.md:41]:
```suggestion
* **selector** specifies the target pods for chaos injection. For more details, see [Define the Scope of Chaos Experiment](experiment_scope.md).
```

**Comment by zhouqiang-cl** [comment]
LGTM

**Pull request merged**